commit 12ee31cacb7ca569ed8c43572af6a49641018352
Author: Tim Otten <totten@civicrm.org>
Date:   Tue Jan 27 22:54:34 2026 -0800

    Hashbang - Prefer bash over sh
    
    In some systems (such as Debian), /bin/sh does -not- default to bash. (It
    defaults to dash, which is a stricter subset.) But I think these scripts
    have been tested mostly against bash.

diff --git a/drupal/scripts/code-clean.sh b/drupal/scripts/code-clean.sh
index 3338b6ada..19aed8419 100644
--- a/drupal/scripts/code-clean.sh
+++ b/drupal/scripts/code-clean.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 find . -name "*~" -type f | xargs rm -f
 find . -name ".#*" -type f | xargs rm -f
diff --git a/drupal/scripts/cron-curl.sh b/drupal/scripts/cron-curl.sh
index 9b168ab36..38a2ad1d2 100644
--- a/drupal/scripts/cron-curl.sh
+++ b/drupal/scripts/cron-curl.sh
@@ -1,3 +1,3 @@
-#!/bin/sh
+#!/bin/bash
 
 curl --silent --compressed http://example.com/cron.php
diff --git a/drupal/scripts/cron-lynx.sh b/drupal/scripts/cron-lynx.sh
index 904667acb..2ae5cab52 100644
--- a/drupal/scripts/cron-lynx.sh
+++ b/drupal/scripts/cron-lynx.sh
@@ -1,3 +1,3 @@
-#!/bin/sh
+#!/bin/bash
 
 /usr/bin/lynx -source http://example.com/cron.php > /dev/null 2>&1
diff --git a/scripts/backup.sh b/scripts/backup.sh
index 9590e185c..5f4f0b899 100755
--- a/scripts/backup.sh
+++ b/scripts/backup.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # backup.sh - Back up Bluebird code and data to a local or non-local directory.
 #
diff --git a/scripts/bluebird_setup.sh b/scripts/bluebird_setup.sh
index d1e130307..0b2be6f02 100755
--- a/scripts/bluebird_setup.sh
+++ b/scripts/bluebird_setup.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # bluebird_setup.sh - Initialize ("spin up") a Bluebird CRM instance.
 #
diff --git a/scripts/changeCollation.sh b/scripts/changeCollation.sh
index 96df2e37a..235c29793 100755
--- a/scripts/changeCollation.sh
+++ b/scripts/changeCollation.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # changeCollation.sh
 #
diff --git a/scripts/changelog_v2/install.sh b/scripts/changelog_v2/install.sh
index afb5abf78..8f35fcbad 100755
--- a/scripts/changelog_v2/install.sh
+++ b/scripts/changelog_v2/install.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # install.sh - Install the new changelog summary/detail layer.
 #
diff --git a/scripts/cleanup_csv.sh b/scripts/cleanup_csv.sh
index 3ac23d2b3..e19403235 100755
--- a/scripts/cleanup_csv.sh
+++ b/scripts/cleanup_csv.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # cleanup_csv.sh - Convert non-printable characters to printable equivalents,
 #                  and eliminate all pipes, backslashes, and tildes.
diff --git a/scripts/clearCache.sh b/scripts/clearCache.sh
index d651bf965..b94ec0755 100755
--- a/scripts/clearCache.sh
+++ b/scripts/clearCache.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # clearCache.sh
 #
@@ -184,7 +184,7 @@ data_owner=`$readConfig --ig $instance data.rootdir.owner | cut -d: -f1` || data
 
 if [ $EUID -eq 0 -a $skip_root_check -ne 1 ]; then
   echo "$prog: Running as root causes file permission problems; restarting as user apache"
-  exec su $data_owner -s /bin/sh -c "$0 $*"
+  exec su $data_owner -s /bin/bash -c "$0 $*"
 fi
 
 
diff --git a/scripts/convertLogInnoDB.sh b/scripts/convertLogInnoDB.sh
index 70e4f7b62..3fd815f73 100755
--- a/scripts/convertLogInnoDB.sh
+++ b/scripts/convertLogInnoDB.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # convertLogInnoDB.sh - Convert logging tables to compressed InnoDB.
 #
diff --git a/scripts/convertUtf8mb4.sh b/scripts/convertUtf8mb4.sh
index 96d56a964..00edfc58b 100755
--- a/scripts/convertUtf8mb4.sh
+++ b/scripts/convertUtf8mb4.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # convertUtf8mb4.sh
 #
diff --git a/scripts/convert_nonprintables.sh b/scripts/convert_nonprintables.sh
index 85f6df057..d08e3315a 100755
--- a/scripts/convert_nonprintables.sh
+++ b/scripts/convert_nonprintables.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # convert_nonprintables.sh - Convert all non-printable characters to
 #                            printable equivalents.
diff --git a/scripts/copyArchive.sh b/scripts/copyArchive.sh
index 335752ad8..957021e2e 100755
--- a/scripts/copyArchive.sh
+++ b/scripts/copyArchive.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # copyArchive.sh - Copy an instance archive file (zip file containing two
 #                  SQL dump files) to a new file, prevserving proper naming.
diff --git a/scripts/copyInstance.sh b/scripts/copyInstance.sh
index 5d0f0fc58..7276df79a 100755
--- a/scripts/copyInstance.sh
+++ b/scripts/copyInstance.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # copyInstance.sh - Copy one CRM database instance into another.
 #
diff --git a/scripts/countDupEmails.sh b/scripts/countDupEmails.sh
index 028dba608..492f9c331 100755
--- a/scripts/countDupEmails.sh
+++ b/scripts/countDupEmails.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # countDupEmails.sh - Provides statistics on number of duplicate e-mails in CRM.
 #
diff --git a/scripts/countIssueNotes.sh b/scripts/countIssueNotes.sh
index 512fcff97..7a236e3a3 100755
--- a/scripts/countIssueNotes.sh
+++ b/scripts/countIssueNotes.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # countIssueNotes.sh - Count number of "OMIS ISSUE CODES" notes in CRM.
 #
diff --git a/scripts/cv.sh b/scripts/cv.sh
index 6ea4f3970..b20b0284f 100755
--- a/scripts/cv.sh
+++ b/scripts/cv.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # cv.sh - A Bluebird-aware wrapper around the CiviCRM "cv" utility.
 #
diff --git a/scripts/dedupeSetup.sh b/scripts/dedupeSetup.sh
index 3738d5650..c086602f3 100755
--- a/scripts/dedupeSetup.sh
+++ b/scripts/dedupeSetup.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # dedupeSetup.sh - All-in-one dedupe setup script.
 #
diff --git a/scripts/defaults.sh b/scripts/defaults.sh
index 31145c6d7..48e6fd2e4 100755
--- a/scripts/defaults.sh
+++ b/scripts/defaults.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # defaults.sh - Shell defaults when using the Bluebird config file.
 #
diff --git a/scripts/deleteBackups.sh b/scripts/deleteBackups.sh
index 973889503..4562ff8a1 100755
--- a/scripts/deleteBackups.sh
+++ b/scripts/deleteBackups.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # deleteBackups.sh - Delete instance backup files (data dumps)
 #
diff --git a/scripts/deleteEmptyEmailPhone.sh b/scripts/deleteEmptyEmailPhone.sh
index d4468b749..8073e83f5 100755
--- a/scripts/deleteEmptyEmailPhone.sh
+++ b/scripts/deleteEmptyEmailPhone.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # deleteEmptyEmailPhone.sh
 #
diff --git a/scripts/deleteEntityFileDupes.sh b/scripts/deleteEntityFileDupes.sh
index d467de427..b2ab29cbd 100755
--- a/scripts/deleteEntityFileDupes.sh
+++ b/scripts/deleteEntityFileDupes.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # deleteEntityFileDupes.sh
 #
diff --git a/scripts/deleteExports.sh b/scripts/deleteExports.sh
index 847b41d5f..6e57715bb 100755
--- a/scripts/deleteExports.sh
+++ b/scripts/deleteExports.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # deleteExports.sh - Delete print production export files
 #
diff --git a/scripts/deleteInstance.sh b/scripts/deleteInstance.sh
index c42fe9b1e..24c78fa37 100755
--- a/scripts/deleteInstance.sh
+++ b/scripts/deleteInstance.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # deleteInstance.sh - Delete the databases and files for a CRM instance
 #
diff --git a/scripts/deleteOrphanedTags.sh b/scripts/deleteOrphanedTags.sh
index b9c482b4f..f46ec3b38 100755
--- a/scripts/deleteOrphanedTags.sh
+++ b/scripts/deleteOrphanedTags.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # deleteOrphanedTags.sh - Delete Bluebird tags that are not associated with
 #                         any entities (contacts, cases, or activities),
diff --git a/scripts/dropCiviTriggers.sh b/scripts/dropCiviTriggers.sh
index 1ff8540cf..1e830012e 100755
--- a/scripts/dropCiviTriggers.sh
+++ b/scripts/dropCiviTriggers.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # dropCiviTriggers.sh
 #
diff --git a/scripts/drush.sh b/scripts/drush.sh
index db6ddeebc..69f9e158f 100755
--- a/scripts/drush.sh
+++ b/scripts/drush.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # drush.sh - A Bluebird-aware wrapper around the "drush" utility.
 #
diff --git a/scripts/filesToUpper.sh b/scripts/filesToUpper.sh
index 87d432a0d..fbe66a804 100755
--- a/scripts/filesToUpper.sh
+++ b/scripts/filesToUpper.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # uppercase any filenames with lowercase chars
 for file in $*
  do
diff --git a/scripts/fixDupAddr.sh b/scripts/fixDupAddr.sh
index c34883cba..f6f5bad77 100755
--- a/scripts/fixDupAddr.sh
+++ b/scripts/fixDupAddr.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # fixDupAddr.sh - Nullify supp1 or supp2 values if they duplicate street_address
 #
diff --git a/scripts/fixDupePrimary.sh b/scripts/fixDupePrimary.sh
index 5b4af4c0a..133a0298a 100755
--- a/scripts/fixDupePrimary.sh
+++ b/scripts/fixDupePrimary.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # fixDupePrimary.sh
 # 
diff --git a/scripts/fixEntityTags.sh b/scripts/fixEntityTags.sh
index 929c11143..d9ff55643 100755
--- a/scripts/fixEntityTags.sh
+++ b/scripts/fixEntityTags.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # fixEntityTags.sh - Analyze the civicrm_entity_tag table and fix problems.
 #
diff --git a/scripts/fixOmisStreetNum.sh b/scripts/fixOmisStreetNum.sh
index 62e78c649..670a1ad47 100755
--- a/scripts/fixOmisStreetNum.sh
+++ b/scripts/fixOmisStreetNum.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # fixOmisStreetNum.sh - Use OMIS saved data to split HOUSE field into
 #                       street_number and street_number_suffix
diff --git a/scripts/fixPermissions.sh b/scripts/fixPermissions.sh
index b0842886a..d53407337 100755
--- a/scripts/fixPermissions.sh
+++ b/scripts/fixPermissions.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # fixPermissions.sh - Set Bluebird directory permissions appropriately.
 #
diff --git a/scripts/fixPhoneTypes.sh b/scripts/fixPhoneTypes.sh
index c3aec0fc4..4d0ee1aa8 100755
--- a/scripts/fixPhoneTypes.sh
+++ b/scripts/fixPhoneTypes.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # fixPhoneTypes.sh - correct phone types assigned during OMIS import
 #
diff --git a/scripts/fixPrefixes.sh b/scripts/fixPrefixes.sh
index 88a38fdee..b6542ea3d 100755
--- a/scripts/fixPrefixes.sh
+++ b/scripts/fixPrefixes.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # fixPrefixes.sh - For records without a prefix, set prefix according to gender.
 #
diff --git a/scripts/fixPvtAddr.sh b/scripts/fixPvtAddr.sh
index 4ea6876e7..a2682006a 100755
--- a/scripts/fixPvtAddr.sh
+++ b/scripts/fixPvtAddr.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # fixPvtAddr.sh - Remove "Pvt" from street_name and street_unit fields.
 #
diff --git a/scripts/fixRecordTypes.sh b/scripts/fixRecordTypes.sh
index aa492eb70..bb0c3e79f 100755
--- a/scripts/fixRecordTypes.sh
+++ b/scripts/fixRecordTypes.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # fixRecordTypes.sh - Set the record_type based on saved OMIS data
 #
diff --git a/scripts/fixSpaces.sh b/scripts/fixSpaces.sh
index e59d0c2c9..6479f6bf2 100755
--- a/scripts/fixSpaces.sh
+++ b/scripts/fixSpaces.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # fixSpaces.sh - Remove extraneous spaces from various databases fields.
 #
diff --git a/scripts/fixSuppAddr1.sh b/scripts/fixSuppAddr1.sh
index c5d270fab..40a6eb764 100755
--- a/scripts/fixSuppAddr1.sh
+++ b/scripts/fixSuppAddr1.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # fixSuppAddr1.sh - Set supplemental_address_1 to NULL wherever possible.
 #
diff --git a/scripts/fixSuppAddr2.sh b/scripts/fixSuppAddr2.sh
index d3b057a07..10aa2d36f 100755
--- a/scripts/fixSuppAddr2.sh
+++ b/scripts/fixSuppAddr2.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # fixSuppAddr2.sh - Move e-mail addresses and/or phone numbers from
 #                   supplemental_address_2 into e-mail/phone tables.
diff --git a/scripts/flagBadEmails.sh b/scripts/flagBadEmails.sh
index e422983df..0304ede9e 100755
--- a/scripts/flagBadEmails.sh
+++ b/scripts/flagBadEmails.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # flagBadEmails.sh - Analyze email addresses for validity
 #
diff --git a/scripts/hitInstance.sh b/scripts/hitInstance.sh
index d775f5c35..b10e15737 100755
--- a/scripts/hitInstance.sh
+++ b/scripts/hitInstance.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # hitInstance.sh
 #
diff --git a/scripts/iterateInstances.sh b/scripts/iterateInstances.sh
index 710797a02..9996e9f24 100755
--- a/scripts/iterateInstances.sh
+++ b/scripts/iterateInstances.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # iterateInstances.sh - Perform a command for one or more CRM instances
 #
@@ -191,7 +191,7 @@ for instance in $instances; do
     [ $quiet_mode -eq 0 ] && logdt "[$run_as@$instance] About to exec: $realcmd $bg_jobs" >&2
     [ $show_timing -eq 1 ] && echo "==> START [`date +%H:%M:%S.%N`]"
     if [ "$run_as" ]; then
-      su $run_as -s /bin/sh -c "$realcmd $bg_jobs"
+      su $run_as -s /bin/bash -c "$realcmd $bg_jobs"
     else
       eval $realcmd $bg_jobs
     fi
diff --git a/scripts/manageAccumulator.sh b/scripts/manageAccumulator.sh
index a8d22d70b..b542618ba 100755
--- a/scripts/manageAccumulator.sh
+++ b/scripts/manageAccumulator.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # manageAccumulator.sh - Perform operations on the Sendgrid Stats Accumulator
 #
diff --git a/scripts/manageCiviConfig.sh b/scripts/manageCiviConfig.sh
index 9907c867f..b7a58a23b 100755
--- a/scripts/manageCiviConfig.sh
+++ b/scripts/manageCiviConfig.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # manageCiviConfig.sh - Wrapper around manageCiviConfig.php
 #
diff --git a/scripts/manageLdapConfig.sh b/scripts/manageLdapConfig.sh
index cf3a0df2f..406ea5737 100755
--- a/scripts/manageLdapConfig.sh
+++ b/scripts/manageLdapConfig.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # manageLdapConfig.sh - Wrapper around manageLdapConfig.php
 #
diff --git a/scripts/manageSolrConfig.sh b/scripts/manageSolrConfig.sh
index 9845f6699..d3085d42c 100755
--- a/scripts/manageSolrConfig.sh
+++ b/scripts/manageSolrConfig.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # manageSolrConfig.sh - Handle setup of SOLR integration
 #
diff --git a/scripts/mercury_install.sh b/scripts/mercury_install.sh
index 10a0ab194..9cd713d74 100755
--- a/scripts/mercury_install.sh
+++ b/scripts/mercury_install.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # mercury_install.sh - Automate the process of installing Pantheon/Mercury
 #
diff --git a/scripts/migrateSurveyData.sh b/scripts/migrateSurveyData.sh
index 9a50e893a..763113a32 100755
--- a/scripts/migrateSurveyData.sh
+++ b/scripts/migrateSurveyData.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # v37.sh
 #
diff --git a/scripts/padAddrInfo.sh b/scripts/padAddrInfo.sh
index 0d726630c..be024aeb2 100755
--- a/scripts/padAddrInfo.sh
+++ b/scripts/padAddrInfo.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # padAddrInfo.sh - Left-pad ZIP+4, School District, and County with zeroes
 #
diff --git a/scripts/readConfig.sh b/scripts/readConfig.sh
index 634fea285..061795af4 100755
--- a/scripts/readConfig.sh
+++ b/scripts/readConfig.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # readConfig.sh - Read the Bluebird config file and return values
 #
diff --git a/scripts/rebuildCachedValues.sh b/scripts/rebuildCachedValues.sh
index 0881cf206..c8daf6845 100755
--- a/scripts/rebuildCachedValues.sh
+++ b/scripts/rebuildCachedValues.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # rebuildCachedValues.sh - Rebuild certain CiviCRM cached values.
 #
diff --git a/scripts/removeCopyright.sh b/scripts/removeCopyright.sh
index 4e583ac5d..249cf2e0e 100755
--- a/scripts/removeCopyright.sh
+++ b/scripts/removeCopyright.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # removeCopyright.sh - Remove any Copyright statement from e-mail footers
 #
diff --git a/scripts/renameInstance.sh b/scripts/renameInstance.sh
index bce4b6291..f8cd9d00c 100755
--- a/scripts/renameInstance.sh
+++ b/scripts/renameInstance.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # renameInstance.sh - Rename a CRM database instance.
 #
diff --git a/scripts/reportWebSignups.sh b/scripts/reportWebSignups.sh
index 0451fc340..3692850a8 100755
--- a/scripts/reportWebSignups.sh
+++ b/scripts/reportWebSignups.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # reportWebSignups.sh - Generate reports (typically scheduled weekly) for
 #                       nysenate.gov web signups and send to Senators.
diff --git a/scripts/resetRolePerms.sh b/scripts/resetRolePerms.sh
index 6e1afb667..9b37ab789 100755
--- a/scripts/resetRolePerms.sh
+++ b/scripts/resetRolePerms.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # resetRolePerms.sh - Reset all roles and permissions for a CRM instance
 #
diff --git a/scripts/rsync_bluebird.sh b/scripts/rsync_bluebird.sh
index cdb723cbe..869f7e546 100755
--- a/scripts/rsync_bluebird.sh
+++ b/scripts/rsync_bluebird.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # rsync_bluebird.sh - Synchronize the local repo with the production codebase
 #
diff --git a/scripts/sageSetup.sh b/scripts/sageSetup.sh
index 318ed5041..b2f6eead7 100755
--- a/scripts/sageSetup.sh
+++ b/scripts/sageSetup.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # sageSetup.sh - The nyss_sage setup script
 #
diff --git a/scripts/setErrorPage.sh b/scripts/setErrorPage.sh
index 0bab1bce3..dfdf5b96f 100755
--- a/scripts/setErrorPage.sh
+++ b/scripts/setErrorPage.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # setErrorPage.sh
 #
diff --git a/scripts/setOnHold.sh b/scripts/setOnHold.sh
index 9b512694d..4e02ee616 100755
--- a/scripts/setOnHold.sh
+++ b/scripts/setOnHold.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # setOnHold.sh - Given a file of undeliverable e-mail addresses, set the
 #                on_hold field for all matching e-mails in the CRM.
diff --git a/scripts/setOptOuts.sh b/scripts/setOptOuts.sh
index c72e53bc9..24ada5931 100755
--- a/scripts/setOptOuts.sh
+++ b/scripts/setOptOuts.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # setOptOuts.sh - Given a file of opted out e-mail addresses, set the
 #                 is_opt_out field for matching contacts.
diff --git a/scripts/setPreferences.sh b/scripts/setPreferences.sh
index 6463867fa..b24b88999 100755
--- a/scripts/setPreferences.sh
+++ b/scripts/setPreferences.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # setPreferences.sh - Set CiviCRM preferences for the administrative user.
 #
diff --git a/scripts/setupCiviMail.sh b/scripts/setupCiviMail.sh
index 6badb1f21..98d63e879 100755
--- a/scripts/setupCiviMail.sh
+++ b/scripts/setupCiviMail.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # setupCiviMail.sh
 #
diff --git a/scripts/shadowTable.sh b/scripts/shadowTable.sh
index b57c8f063..37aabc9e0 100755
--- a/scripts/shadowTable.sh
+++ b/scripts/shadowTable.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # shadowTable.sh
 #
